kore/src/Kore/Step/Axiom/EvaluationStrategy.hs

```diff
@@ -238,11 +238,21 @@ applyFirstSimplifierThatWorks
               -- than one result, so we throw an error.
               error
                   (  "Unexpected simplification result with more "
-                  ++ "than one configuration: "
-                  ++ show applicationResult
+                  ++ "than one configuration: \n"
+                  ++ (show . Pretty.vsep $
```

Comment:
should i use `Pretty.renderStrict . Pretty.layoutCompact` before `Pretty.vsep` like the code below?

Reply:
No, `show` should be fine.